xbmc/cores/DllLoader/exports/util/EmuFileWrapper.cpp

```diff
@@ -73,17 +77,11 @@ void CEmuFileWrapper::CleanUp()
         delete m_files[i].file_lock;
         m_files[i].file_lock = nullptr;
       }
-#if !defined(TARGET_WINDOWS)
       //Don't memset on Windows as it overwrites our pointer
```

Comment:
yes, I see what it does. 

the ifdef was removed, and the comment seems to apply to why the ifdef was there. So if you removed the ifdef, seems the comment should also get removed ? or moved up into new and changed to reflect why that bit of code is there.